test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
```

Comment:
Yeah composition is awkward once we have a 1->N relationship between multiple pod+node mutations,  but i think the slice is also awkward.  sebastian isn't currently experimenting deeply with scheduler predicates per internal conv.